src/main/scala/mesosphere/marathon/upgrade/TaskReplaceActor.scala

```diff
@@ -19,19 +23,23 @@ class TaskReplaceActor(
     driver: SchedulerDriver,
     taskQueue: LaunchQueue,
     taskTracker: TaskTracker,
+    healthCheckManager: HealthCheckManager,
     eventBus: EventStream,
     app: AppDefinition,
     promise: Promise[Unit]) extends Actor with ActorLogging {
   import context.dispatcher
 
-  val tasksToKill = taskTracker.get(app.id)
+  private[this] val log = LoggerFactory.getLogger(getClass)
```

Comment:
We actually don't need this, since TaskReplaceActor uses the ActorLogging mixin.